### DISCUSSION TITLE
Making JzAzBz gradients

### DISCUSSION THREAD

**Discussion by P-Star7** [ImageMagick/ImageMagick#5759] [Help]
I tried to generate a gradient using JzAzBz colourspace by using the instructions here:
[https://legacy.imagemagick.org/Usage/canvas/#gradient_colorspace](url)
`convert -size 1x240 gradient: "#0000FF"-"#FFFFFF" -set colorspace JzAzBz -colorspace sRGB gradient.png`
However, the result is as you see:
![gradient2](https://user-images.githubusercontent.com/26371167/202322997-1593311b-78dd-4dac-ade5-08fdd86e3845.png)
It looks glitchy. Is this because I used RGB colour notation as the gradient's colours? If so, what is "proper" JzAzBz notation?
I won't post this as an issue yet, because it's quite possible that this is just a user error on my part.

**Comment by snibgo** [comment]
What version of IM, on what platform?

Your command fails for me, because it has a bug.

Most JzAzBz tuples are outside the sRGB gamut.
